Siv3D/include/Siv3D/Geometry2D.hpp

```diff
@@ -2762,6 +2762,56 @@ namespace s3d
 		[[nodiscard]]
 		Polygon ConcaveHull(const Vec2* points, size_t size, double concavity = 2.0, double lengthThreshold = 0.0);
 
+		//////////////////////////////////////////////////
+		//
+		//	SmallestEnclosingCircle
+		//
+		//////////////////////////////////////////////////
+
+		/// @brief 点 p0, p1, p2 の最小包含円を返します。
+		/// @param p0 点 0
+		/// @param p1 点 1
+		/// @param p2 点 2
+		/// @returns 点 p0, p1, p2 の最小包含円
```

Comment:
```suggestion
		/// @return 点 p0, p1, p2 の最小包含円
```

`@return` か `@returns` かは統一したほうが美しい。